api/handlers.go

```diff
@@ -656,6 +657,43 @@ func (api *API) userPUT(u *database.User, w http.ResponseWriter, req *http.Reque
 	api.loginUser(w, u, true)
 }
 
+// userPubKeyDELETE removes a given pubkey from the list of pubkeys associated
+// with this user. It does not require a challenge-response because the used
```

Comment:
Think it should be this, unless it was meant as an homage to Stallman.

```suggestion
// with this user. It does not require a challenge-response because the user
```